pkg/tsdb/service.go

```diff
@@ -70,8 +77,12 @@ func (s *Service) HandleRequest(ctx context.Context, ds *models.DataSource, quer
 
 		return plugin.DataQuery(ctx, ds, query)
 	}
+	handler := backend.QueryDataHandler(s.BackendPluginManager)
```

Comment:
perhaps there is a way to shove a statically defined plugin into the `BackendPluginManager`?